packages/kit/test/apps/basics/test/cross-platform/test.js

```diff
@@ -962,6 +962,16 @@ test.describe('Routing', () => {
 			expect(await page.textContent('h1')).toBe('symlinked');
 		});
 	}
+	test('__data.json - trailing slash always', async ({ request }) => {
```

Comment:
the other tests all have a blank line before them. let's keep that up for consistency
```suggestion

	test('__data.json - trailing slash always', async ({ request }) => {
```

Reply:
@benmccann 

1. line break is corrected
2. the trailing slash config can be found here: [packages/kit/test/apps/basics/src/routes/routing/trailing-slash-ssr/always/+page.js ](https://github.com/sveltejs/kit/pull/9738/files#diff-2482dd9e0a13c821bb9f715f83ce9bed6f7da31b49ef39deebcf98381180e1ac)

The thing I'm trying to test, is the correct trailing slash in the pathname in `__data.json`, as it currently always falls back to `never`. (see https://github.com/sveltejs/kit/issues/9595#issuecomment-1516024603)